ccf_transaction_processor/scripts/configure_ccf_network.py

```diff
@@ -147,8 +194,11 @@ def Main() :
         LOG.error('failed to connect to CCF service')
         sys.exit(-1)
 
-    open_network_script(member_client, options, config)
-    add_user_script(member_client, options, config)
+    if options.add_node:
+        add_node_script(member_client, options, config)
+    else:
+        open_network_script(member_client, options, config)
+        add_user_script(member_client, options, config)
```

Comment:
suggestion... require a command parameter since you are really executing different flows. the commands could be consistent with the ones for start_cchost.sh (start and join).